helm/alfresco-content-services/README.md

```diff
@@ -15,11 +15,14 @@ A Helm chart for deploying Alfresco Content Services
 | Repository | Name | Version |
 |------------|------|---------|
 |  | activemq | 2.0.0 |
+|  | alfresco-elasticsearch-connector | 0.1.0 |
```

Comment:
Is it possible to make this version match the version of ESC?  That would seem easier to understand.